lib/SystemStatistics.php

```diff
@@ -94,4 +95,25 @@ protected function getMemoryUsage() {
 		];
 	}
 
+	/**
+	 * Get current CPU load average
+	 *
+	 * @return array load average with three values, 1/5/15 minutes average.
+	 */
+	protected function getProcessorUsage() {
+		// get current system load average.
+		$loadavg = sys_getloadavg();
+
+		// check if we got any values back.
+		if (!(is_array($loadavg) && count($loadavg) === 3)) {
+			// either no array or too few array keys.
+			// returning back zeroes to prevent any errors on JS side.
+			$loadavg = [0, 0, 0];
```

Comment:
wonder if it would make more sense to return a error code and then hide the CPU Load in the web interface or display something like "not available". Otherwise it looks like everything is OK and the system load is just super low, something which makes probably every admin happy. 😉